src/hyperion/external_interaction/callbacks/xray_centre/callback_collection.py

```diff
@@ -41,3 +48,23 @@ def from_params(cls, parameters: InternalParameters):
             zocalo_handler=zocalo_handler,
         )
         return callback_collection
+
+
+@dataclass(frozen=True, order=True)
+class VmxmFastGridScanCallbackCollection(AbstractPlanCallbackCollection):
```

Comment:
See [my comment of the equivalent bit of code for `i03`](https://github.com/DiamondLightSource/hyperion/pull/942/files/35ee736c34742df5a55882a3ea05c57110cdeaac#r1404226961):

> Still feels like we should be passing this from "higher up" the call chain. But probably fine for now. Possibly create an issue to think about where this comes from.

So yes I agree with you :)

Reply:
Don't put _too_ much effort into this though, because I think David (or Dom?) is looking at pulling all or most of these callbacks into ophyd devices which would then make this obsolete anyway.